<launch>
    <rosparam command="load" file="$(find infantry)/config/camera.yaml" ns="camera" />
    <node pkg="infantry" type="infantry_node" name="infantry_node" respawn="true" output="screen" /> 
</launch>